### PR TITLE
[doc] Add example of using WS from main

### PR DESCRIPTION
**Pull request by wsargent**
Adds a description of using Play WS from a main method in the simple case, with no bells or whistles.

### CODE REVIEW

File: documentation/manual/working/javaGuide/main/ws/code/javaguide/ws/Standalone.java [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+                .setShutdownTimeout(0).build();
+
+        String name = "wsclient";
+        ActorSystem system = ActorSystem.create(name);
+        ActorMaterializerSettings settings = ActorMaterializerSettings.create(system);
+        ActorMaterializer materializer = ActorMaterializer.create(settings, system, name);
+
+        WSClient client = new AhcWSClient(config, materializer);
+        client.url("http://www.google.com").get().whenComplete((r, e) -> {
+            Optional.of(r).ifPresent(response -> {
```
Comment (gmethvin) [inline documentation/manual/working/javaGuide/main/ws/code/javaguide/ws/Standalone.java]: Shouldn't we use `Optional.ofNullable` here?

File: documentation/manual/working/scalaGuide/main/ws/code/ScalaWSStandalone.scala [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+    }
+  }
+
+  def call(wsClient: WSClient): Future[Unit] = {
+    wsClient.url("http://www.google.com").get().map { response =>
+      println("Got a response!")
+      println(response.allHeaders)
+    }.andThen {
+      case _ =>
+        wsClient.close()
```
Comment (gmethvin) [inline documentation/manual/working/scalaGuide/main/ws/code/ScalaWSStandalone.scala]: We should probably do this in `main`. The way it's structured now makes it look like `close` should happen on every call.

File: documentation/manual/working/javaGuide/main/ws/code/javaguide/ws/Standalone.java (hunk 1 above)
Comment (wsargent) [inline documentation/manual/working/javaGuide/main/ws/code/javaguide/ws/Standalone.java]: ...wow.  It did not occur to me `Optional.of` was not the same as `Option()`.

File: documentation/manual/working/scalaGuide/main/ws/code/ScalaWSStandalone.scala (hunk 2 above)
Comment (wsargent) [inline documentation/manual/working/scalaGuide/main/ws/code/ScalaWSStandalone.scala]: Done.

**Comment by wsargent** [comment]
Needs backport to 2.5.x

**Pull request merged**